blackbox/docs/general/ddl/sharding.rst

```diff
@@ -46,8 +46,10 @@ is applied, (see :ref:`ref_clustered_clause`).
 
 .. NOTE::
 
-   The number of shards can only be set on table creation, it cannot be changed
-   later on.
+   The number of shards can be changed later on but it involves having the table
+   in a ``READ-ONLY`` state until the operation is completed
+   (see :ref:`alter_change_number_of_shard`).
+   So, it is better to tune this carefully in advance.
```

Comment:
"So, it is better to tune this carefully in advance." - maybe it's a personal preference, but I find this a bit condescending. I wouldn't keep it in the documentation.